src/board/system76/common/acpi.c

```diff
@@ -133,8 +134,13 @@ uint8_t acpi_read(uint8_t addr) {
             break;
 #endif // HAVE_LED_AIRPLANE_N
 
-        // Set size of flash (from old firmware)
-        ACPI_8 (0xE5, 0x80);
+        // S0ix hook
+        case 0xE0:
+            data = pep_in_s0ix;
+            break;
+        case 0xE1:
+            data = pep_display_on;
+            break;
```

Comment:
Why not bit field? We could use a single byte here with two bits

Reply:
Switched to bitfields here and in https://github.com/Dasharo/coreboot/pull/433